docs/content/docs/dev/datastream/operators/overview.md

```diff
@@ -755,5 +755,43 @@ someStream.filter(...).slotSharingGroup("name")
 ```python
 some_stream.filter(...).slot_sharing_group("name")
 ```
+
+## Name And Description
+Operators and job vertices in flink have a name and a description. 
+Both name and description are introduction about what an operator or a job vertex is doing, but they are used differently.
+
+The name of operator and job vertex will be used in web ui, thread name, logging, metrics, etc.
+The name of a job vertex is constructed based on the name of operators in it.
+The name need to be as concise as possible to avoid big pressure on external systems.
+
+The description will be used in the execution plan and displayed as details of a job vertex in web UI.
+The description of a job vertex is constructed based on the description of operators in it.
+The description can contain detail information about operators to facilitate debugging at runtime.
+
+{{< tabs slotsharing >}}
+{{< tab "Java" >}}
+```java
+someStream.filter(...).setName("filter").setDescription("x in (1, 2, 3, 4) and y > 1")
+```
+{{< /tab >}}
+{{< tab "Scala" >}}
+```scala
+someStream.filter(...).setName("filter").setDescription("x in (1, 2, 3, 4) and y > 1")
+```
+{{< /tab >}}
+{{< tab "Python" >}}
+```python
+some_stream.filter(...).name("filter").set_description("x in (1, 2, 3, 4) and y > 1")
+```
 {{< /tab >}}
 {{< /tabs>}}
+
+The format of description of a job vertex would be a tree format string by default. 
+Users can set `pipeline.vertex-description-mode` to be `CASCADING`, if they want to set description to be cascading format as what it is in former versions.
```

Comment:
I think might change  `set ... to be` to `set ... to`.

`to be cascading format` -> `to be the cascading format`
`as what it is in former versions.` -> `as in former versions.`


Might change to 

Users can set `pipeline.vertex-description-mode` to `CASCADING` if they want to use cascading format description as in former versions.

Similar to `Users can set `table.optimizer.simplify-operator-name-enabled` to be `false`, if they want to set name to be the detailed description as what it is in former versions.`